### PR TITLE
Add openHAB (#6584)

### LINKED ISSUE #6584: Request: OpenHAB
### Brand Name

OpenHAB

### Website

https://www.openhab.org/

### Popularity Metric

The Alexa rank is 40,433 see https://www.alexa.com/siteinfo/openhab.org
https://github.com/openhab

### Official Resources for Icon and Color

image
https://www.openhab.org/about/logos.html

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by mu88**
**Issue:** closes #6584

**Similarweb rank:** [#111,816](https://www.similarweb.com/website/openhab.org/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by dushmanta05** [comment]
Hi there! I'm facing same issues, If you resolve your issue please let me know, Thanks.

**Comment by dushmanta05** [comment]
Hi! I've resolved my issue (got help from someone), I would like to help you in this.

**Comment by mu88** [comment]
@dushmanta05 Thx, I really appreciate it if you'd join me solving this issue 💪🏻

File: icons/openhab.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10.703.148c-.695.07-1.168.168-1.957.387C4.95 1.59 1.786 4.665.606 8.47c-.422 1.386-.5 1.902-.5 3.566 0 1.555.07 2.133.394 3.238l.148.489.77-.77.762-.77-.106-.597c-.117-.715-.14-2.29-.027-3.035a9.96 9.96 0 0 1 2.98-5.762c1.856-1.793 4.008-2.719 6.586-2.832 1.102-.039 1.856.035 2.895.297 3.773.953 6.676 4.094 7.41 8.012.168.906.176 2.48.008 3.425-.383 2.137-1.39 4.008-2.98 5.54-1.805 1.738-4 2.69-6.489 2.804-2.781.133-5.348-.847-7.426-2.832l-.734-.699-.668.664-.664.664.48.48c.961.981 1.953 1.708 3.192 2.337 3.625 1.847 7.918 1.707 11.468-.391 2.38-1.395 4.399-3.91 5.258-6.543.446-1.375.524-1.934.524-3.719.008-1.433-.016-1.707-.157-2.363a11.96 11.96 0 0 0-1.07-3.008A11.927 11.927 0 0 0 10.703.148ZM6.637 11.824l-5.324 5.32.199.41c.105.227.324.61.48.856l.29.45 9.753-9.755 3.707 3.707 3.7 3.711.16-.265c.093-.149.277-.551.418-.899l.246-.632-4.114-4.118C13.882 8.344 12.02 6.492 12 6.492c-.02 0-2.434 2.403-5.363 5.332Zm0 0"/></svg>
```
Comment (PratyushJayachandran) [inline icons/openhab.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>openHAB</title><path d="M10.7058.0708c-.695.0703-1.168.1687-1.957.3886-3.796 1.0593-6.96 4.147-8.14 7.9675-.422 1.3917-.5 1.9098-.5 3.5806 0 1.5614.07 2.1417.394 3.2513l.148.491.77-.7732.762-.7731-.106-.5995c-.117-.718-.14-2.2994-.027-3.0474a10.0146 10.0146 0 0 1 2.98-5.7856c1.856-1.8004 4.008-2.7302 6.586-2.8436 1.102-.0392 1.856.0351 2.895.2982 3.773.9569 6.676 4.1108 7.41 8.0448.168.9097.176 2.4902.008 3.439-.383 2.1458-1.39 4.0245-2.98 5.5627-1.805 1.7452-4 2.701-6.489 2.8155-2.781.1336-5.348-.8504-7.426-2.8436l-.734-.7018-.668.6667-.664.6667.48.482c.961.985 1.953 1.715 3.192 2.3465 3.625 1.8546 7.918 1.714 11.468-.3926 2.38-1.4007 4.399-3.926 5.258-6.5698.446-1.3806.524-1.9419.524-3.7342.008-1.4389-.016-1.714-.157-2.3727a12.0414 12.0414 0 0 0-1.07-3.0203c-2.232-4.476-7.001-7.086-11.957-6.5437zm-4.066 11.7238-5.324 5.3418.199.4117c.105.228.324.6125.48.8595l.29.4519 9.753-9.795 3.707 3.7222 3.7 3.7262.16-.2661c.093-.1496.277-.5533.418-.9027l.246-.6346-4.114-4.1348c-2.27-2.2743-4.132-4.1339-4.152-4.1339s-2.434 2.4128-5.363 5.3538z"/></svg>
```

SVG fixed to specs

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8077,6 +8077,11 @@
             "hex": "5586A4",
             "source": "https://www.khronos.org/legal/trademarks/"
         },
+        {
+            "title": "openHAB",
+            "hex": "000000",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:8082]:
```suggestion
            "hex": "E64A19",
```
Pick dominant color from logo

File: _data/simple-icons.json [hunk 3]
```diff
@@ -8077,6 +8077,11 @@
...
+            "source": "https://www.openhab.org/artwork.html/"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:8083]:
```suggestion
            "source": "https://www.openhab.org/artwork.html/",
```

File: _data/simple-icons.json [hunk 4]
```diff
@@ -8077,6 +8077,11 @@
...
+        },
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:8084]:
```suggestion
            "guidelines": "https://www.openhab.org/about/trademark.html"
        },
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please merge my suggestions or fix the logo by adhering to contribution guidelines

**Comment by PratyushJayachandran** [comment]
> Please merge my suggestions or fix the logo by adhering to contribution guidelines

Preview:
openhab

File: icons/openhab.svg
Comment (adamrusted) [inline icons/openhab.svg:1]: I'm seeing [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/c70f597c-8569-4388-b00c-d79dda68d2e3) between the PR in black and the source file (openHAB_appicon.svg) in red. Could you take another look at this, or let me know which source file you've used please @mu88?

File: _data/simple-icons.json [hunk 5]
```diff
@@ -8218,6 +8218,12 @@
             "hex": "5586A4",
             "source": "https://www.khronos.org/legal/trademarks/"
         },
+        {
+            "title": "openHAB",
+            "hex": "E64A19",
+            "source": "https://www.openhab.org/artwork.html/",
```
Comment (adamrusted) [inline _data/simple-icons.json:8224]:
```suggestion
            "source": "https://www.openhab.org/artwork.html",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/openhab.svg
Comment (mu88) [reply icons/openhab.svg:1]: I've used this one, @adamrusted : https://commons.wikimedia.org/wiki/File:OpenHAB_logo.svg

Comment (PratyushJayachandran) [reply icons/openhab.svg:1]: @adamrusted The suggested changes I made were from https://www.openhab.org/artwork.html and not from Wikimedia which seems to be different from source website.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/bda313b2-d502-43e6-94ae-5bfa31cab622) between the PR icon in black and the `OpenHAB_appicon.svg` file from the artwork page. @mu88 and @PratyushJayachandran could you have a nosey at this?

**Review by adamrusted — APPROVED** [review]

**Pull request merged**